src/benchmark.cc

```diff
@@ -781,7 +798,14 @@ void RunBenchmark(const benchmark::internal::Benchmark::Instance& b,
         report.cpu_accumulated_time = cpu_accumulated_time;
         report.bytes_per_second = bytes_per_second;
         report.items_per_second = items_per_second;
+        report.arg1 = b.arg1;
```

Comment:
are you using arg1 and arg2 anywhere in the reporters? i can't see them at first glance.